OrbitGl/EventTrack.cpp

```diff
@@ -75,6 +75,8 @@ void EventTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingM
 
   const Color kWhite(255, 255, 255, 255);
   const Color kGreenSelection(0, 255, 0, 255);
+  const CaptureData* capture_data = time_graph_->GetCaptureData();
+  CHECK(capture_data);
```

Comment:
Should probably be `capture_data != nullptr` for clarity, same as we agreed on in if-statements. Same in a lot of other places where you access `GetCaptureData()`.

Reply:
Done, thanks.